src/commands/which.ts

```diff
@@ -6,7 +6,7 @@ export default class Which extends Command {
 
   async run(): Promise<void> {
     const {argv} = await this.parse(Which)
-    const cmd = this.config.findCommand(argv.join(':'), {must: true})
+    const cmd = this.config.findCommand(argv.join(this.config.topicSeparator), {must: true})
```

Comment:
I don't think this is what you want to do here.

`findCommand` assumes that is has a normalized id, i.e. it's `:` separated.

Also, is this plugin actually not working for space separated commands? I just installed it in sf and it's working properly as far as I can tell:

```
~ » sf which config get
=== config:get

plugin: @salesforce/plugin-settings
```


```
~ » sf which deploy metadata
=== deploy:metadata

plugin: @salesforce/plugin-deploy-retrieve
```

Reply:
It doesn't work if you put the command in quotes
![image](https://user-images.githubusercontent.com/1084688/199577574-6f24f3bc-d05d-434c-9f6d-e1fbcee8ea2d.png)

But I think you're right that this change isn't doing what I thought it would do.